app/services/NicCalculator.scala

```diff
@@ -73,53 +73,63 @@ trait NicCalculator extends FurloughCapCalculator with CommonCalculationService
       case Nonpayable => Amount(0.00)
     }
 
-    PartialPeriodNicBreakdown(grant, topUp.defaulted, additionalPayment.defaulted, payment)
+    val cappedGrant = nicGrantCap(furloughGrant, grant)
+
+    PartialPeriodNicBreakdown(cappedGrant, topUp.defaulted, additionalPayment.defaulted, payment)
```

Comment:
the `nicGrantCap` can straightaway return the `PartialPeriodNicBreakdown` as same logic is used at both places..
no huge gain but just to avoid code duplication